system-variables.md

```diff
@@ -2377,6 +2377,14 @@ explain select * from t where age=5;
 - Default value: `OFF`
 - This variable is used to control whether to allow `INSERT`, `REPLACE`, and `UPDATE` statements to operate on the `_tidb_rowid` column. This variable can be used only when you import data using TiDB tools.
 
+### tidb_opt_force_inline_cte <span class="version-mark">New in v6.3.0</span>
+
+- Scope: SESSION | GLOBAL
+- Persists to cluster: Yes
+- Type: Boolean
+- Default value: `OFF`
+- This variable is used to control whether CTEs in the entire session are force inlined or not. The default value is `OFF`, which means that inline cte is not enabled by default. However, you can still enable inline cte by specifying `MERGE（）hint`. If the variable is set to `ON`, all CTEs (except recursive CTE) for this session are forced to enable inlining.
```

Comment:
`MERGE（）hint` -> `MERGE()` hint

Reply:
This variable is used to control whether CTEs in the entire session are inlined or not. The default value is `OFF`, which means that inlining CTE is not enforced by default. However, you can still inline CTE by specifying `MERGE()` hint. If the variable is set to `ON`, all CTEs (except recursive CTE) for this session are forced to be inlined.